production/helm/promtail/values.yaml

```diff
@@ -124,3 +124,10 @@ config:
   target_config:
     # Period to resync directories being watched and files being tailed
     sync_period: 10s
+
+service:
+  port: 3101
```

Comment:
is this required ? or could it be `{{ .Values.config.server.http_listen_port }}` ?

Reply:
Nope, it's not. I was in two minds about re-using the config value or adding a service to match: https://github.com/grafana/loki/blob/master/production/helm/loki/values.yaml#L142-L147

I'll rebase to use the former